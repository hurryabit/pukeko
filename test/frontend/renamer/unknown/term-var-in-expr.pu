f : a -> a
f x = g x
