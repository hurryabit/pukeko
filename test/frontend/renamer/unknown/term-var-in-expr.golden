test/frontend/renamer/unknown/term-var-in-expr.pu:2:7: unknown variable: g
